src/stores/domain/general.ts

```diff
@@ -1,37 +1,57 @@
-import { observable, action } from 'mobx'
+import NetInfo from '@react-native-community/netinfo'
+import { action, observable } from 'mobx'
+import { Platform } from 'react-native'
 
-import InputsModel from 'src/models/InputsModel'
+import { ILanguageData, InputsModel } from 'src/models/InputsModel'
 
-interface Data {
-  [key: string]: string
-  languageSource: string
-  languageTranslated: string
+const checkConnectivity = () => {
```

Comment:
And where is this logic used? No meaningful usage whatsoever.

Reply:
![Annotation 2019-05-14 214049](https://user-images.githubusercontent.com/12101306/57723382-2c796700-7691-11e9-809c-8e8e3bc3eebf.png)

Reply:
And wtf is this crap? Why is it not connected to the application life cycle?